src/main/components/form/validator.ts

```diff
@@ -349,6 +349,24 @@ export const isValidCurrency: Validator = value => {
   return 'invalidCurrency';
 };
 
+export const isValidPay: Validator = value => {
```

Comment:
You haven't added any tests for isValidPay (at least not directly in validator.test.ts). Also, can you not repurpose the already tested isValidCurrency?

Reply:
IsValidPay is actually what isValidCurrency was before changes were applied in RET-1929. Changes in RET-1929 was a breaking change actually caused error messages in the PayController to stop being generated as it has different validation rules. It should've been covered by tests but it wasn't. NewJobPayController needed the same error messages as PayController. Rather than repurpose isValidCurrency so it worked for all three controllers(Compensation, Pay, NewJobPay) I reintroduced the old version of IsValidCurrency, renamed IsValidPay, so no regression is required. Tests are now on each controller to confirm each one is generating the correct error messages.

Reply:
I have added some IsValidPay tests to the validator.test.ts though now